pyk/src/pyk/integration_tests/test_kore_to_kast.py

```diff
@@ -0,0 +1,29 @@
+from ..kast import KApply
+from ..kore.syntax import DV, App, SortApp, String
+from ..ktool import KompileBackend
+from ..prelude import intToken
+from .kprove_test import KProveTest
+
+
+class KoreToKastTest(KProveTest):
+    KOMPILE_MAIN_FILE = 'k-files/simple-proofs.k'
+    KOMPILE_BACKEND = KompileBackend.HASKELL
+    KOMPILE_OUTPUT_DIR = 'definitions/simple-proofs'
+    KOMPILE_EMIT_JSON = True
+
+    KPROVE_USE_DIR = '.simple-proof-test'
+
+    @staticmethod
+    def _update_symbol_table(symbol_table):
+        pass
+
+    def test_kast_to_kore(self):
+        kore_kast_pairs = (
+            (App('inj', [SortApp('SortBool'), SortApp('SortKItem')], [App('Lblpred1', [], [DV(SortApp('SortInt'), String('3'))])]), KApply('pred1', [intToken(3)])),
+        )
+        for i, (kore, kast) in enumerate(kore_kast_pairs):
+            with self.subTest(i=i):
```

Comment:
Consider naming each `subTest`. This is important for documenting the intention of each test, and what aspect of the functionality we're testing.

```suggestion
        tests = (
            ('injections',
             App('inj', [SortApp('SortBool'), SortApp('SortKItem')], [App('Lblpred1', [], [DV(SortApp('SortInt'), String('3'))])]),
             KApply('pred1', [intToken(3)])
            ),
        )
        for i, (name, kore, kast) in enumerate(kore_kast_pairs):
            with self.subTest(name, i=i):
```

Reply:
We don't name subtests anywhere else, and I am not convinced it's a good idea. I haven't had any problems finding the problematic subtests in other examples.

I also am just trying to set up the testing harness here, so that as we discover bugs we have a fast way to make tests for them which does not involve invoking the entire codebase or making a new harness. So I'm not trying to make it perfect this time around.

This test, for example, has no sensible name. It's just the first breakage I ran into trying to use `kast_to_kore` and `kore_to_kast`, so I added it as a test.

Reply:
I think `issue-1234` (or whatever the issue number is) is a good name. Not having names makes refactoring the test suite difficult later, because we don't know the intention of the tests.

Reply:
Not worth blocking a merge over.

Reply:
Done.